.gitlab-ci.yml

```diff
@@ -771,86 +801,92 @@ library:ci-fiat_crypto_ocaml:
   extends: .ci-template
   stage: stage-5
   needs:
-  - build:edge+flambda
+  - build:edge+flambda:no-native
```

Comment:
The diff could be quite smaller if you kept the old alias as a default for `no-native`